main.go

```diff
@@ -211,10 +211,11 @@ func setupReconcilers(base utils.ReconcilerBase, mgr manager.Manager) {
 		setupLog.Error(err, "unable to create controller", "controller", "CloudStackMachine")
 		os.Exit(1)
 	}
-	if err := (&controllers.CloudStackMachineStateCheckerReconciler{ReconcilerBase: base}).SetupWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create controller", "controller", "CloudStackMachineStateChecker")
-		os.Exit(1)
-	}
+	// Temporarily disabling machine state checker until we can solve the issue of machines not always being replaced.
```

Comment:
Please remove the commented code because we can always restore the removed code from commit history.